commands_to_load/Mod.py

```diff
@@ -82,5 +83,398 @@ def __init__(self, bot):
         if eObj is not False:
             await ctx.send(embed=eObj)
 
+    @commands.command(aliases=['propm'])
+    async def propagatemute(self, ctx):
+        """Ensures all channels have the muted role as part of its permissions.""" 
+        ## Avoid the admin category of channels and rules and announcments channel since nobody can talk in there anyway
+
+        logger.info('[Mod propagatemute()] propagatemute function detected by user ' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod propagatemute()] invoking message deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod propagatemute()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')

```

Comment:
See this line and all other identical lines, `MUTED_ROLE` can be none. Perhaps explore creating the Muted role if it does not already exist, either on command execution or on startup, or enforcing the precondition that the Muted role must exist already created manually on the server, by outputting some visible error message if it doesn't.

Reply:
@TitanVJ I concur, do a check to ensur that `MUTED_ROLE` does exist and return an error if it doesn't